cmd/bucket-policy.go

```diff
@@ -183,6 +185,21 @@ func getConditionValues(r *http.Request, lc string, username string, claims map[
 			}
 		}
 	}
+	// Add groups claim which could be a list. This will ensure that the claim
+	// `jwt:groups` works.
+	if grpsVal, ok := claims["groups"]; ok {
```

Comment:
@donatello is there any reason why we are doing this only for groups ? it looks like we can do this for all claims

Reply:
We already do this for others, groups was missing @vadmeste